lib/actions/reviewer-survey/index.js

```diff
@@ -0,0 +1,17 @@
+import { laboratoriaAPIAction } from '../helpers';
+
+export const getLastestVersion = () => laboratoriaAPIAction({
+  type: 'REVIEWER_SURVEY_LATEST_VERSION',
+  url: '/reviewer-survey',
+  method: 'GET',
+  key: 'reviewer-survey-latest-version',
```

Comment:
We're already using a syntax like this for store keys:

```suggestion
  key: 'reviewer-survey/latest',
```

As seen on: https://github.com/Laboratoria/api-client/blob/master/lib/actions/users/index.js